lib/zuora/rest.rb

```diff
@@ -3,6 +3,9 @@ module Zuora
   module Rest
     API_URL = 'https://api.zuora.com/rest/v1/'.freeze
     SANDBOX_URL = 'https://apisandbox-api.zuora.com/rest/v1/'.freeze
+    # Newest endpoints for REST API calls
+    BETA_API_URL = 'https://rest.zuora.com/v1/'.freeze
+    BETA_SANDBOX_URL = 'https://rest.apisandbox.zuora.com/v1/'.freeze
```

Comment:
@thelostspore Thanks for having a look.  I agree with both of your points.  I most likely will not change the second point as it will break backwards compatibility.  I do think this should be controlled at a per-request level like you suggest, but the intention of this PR is to keep it backwards compatible.  Changing functionality like that will need a larger version change.  As for the naming around Beta, I agree, but it should just be called API_URL, the former called 'DEPRECATED' or such.